### PR TITLE
Fix syntax error and fix styleguide compliance

### PR DESCRIPTION
**Pull request by kallies**
I've fixed some styleguide compliance issues reported by puppet-lint. The Require syntax issue is fixed, too.

### CODE REVIEW

**Comment by ngiger** [comment]
Personally I no longer use puppet and switched to use salt. But if you want to improve puppet-x2go I am willing to put a notice on github or puppet hub to direct newcomers to your site. I think there is no other way to pass a puppet module to another leader.

**Comment by kallies** [comment]
A notice on Puppetforge would be helpful. You could also delete the module in the [forge](https://forge.puppet.com/) and just keep the repo in GitHub so user would only find the active x2go module(s).

The fork is now active as [fischassetmanagement/x2go](https://forge.puppet.com/fischassetmanagement/x2go).

File: manifests/client.pp [hunk 1]
```diff
@@ -4,12 +4,13 @@
 # under the terms of the GNU General Public License version 3 as published by
 # the Free Software Foundation.class
 class x2go::client (
-  $ensure             =  false,
+  $ensure         = 'present',
+  $install_client = ::x2go::install_client,
```
Comment (kallies) [inline manifests/client.pp]: missing $

File: manifests/server.pp [hunk 2]
```diff
@@ -4,28 +4,12 @@
 # under the terms of the GNU General Public License version 3 as published by
 # the Free Software Foundation
 class x2go::server (
-  $ensure             =  false,
+  $ensure         = 'present',
+  $install_server = ::x2go::install_server,
```
Comment (kallies) [inline manifests/server.pp]: missing $

**Pull request closed without merging**